src/_utils/dayjs-ssr-helper/duration.ts

```diff
@@ -0,0 +1,20 @@
+import * as _duration from 'dayjs/plugin/duration.js';
```

Comment:
same, remove this also

Reply:
Removed deprecated dayjs helper files `duration.ts`